src/config.h.cmake.in

```diff
@@ -188,9 +188,9 @@
 /* Use SOA version of AA distance table */
 #cmakedefine ENABLE_SOA  @ENABLE_SOA@
 
-#if (__cplusplus >= 201103L)
+#if defined(__cplusplus)
   #if defined(__INTEL_COMPILER)
```

Comment:
Why is special handling for the Intel compiler needed here? Are we hurting other compilers with inequivalent setting or working around an implementation gap?

Reply:
Giving compiler information about alignment is not portable across compilers.
GNU/Clang use __builtin_assume_aligned
Intel uses __assume_aligned
This is the only difference here.